pkg/api/login_oauth.go

```diff
@@ -17,9 +23,9 @@ import (
 )
 
 func GenStateString() string {
-        rnd := make([]byte, 32)
-        rand.Read(rnd)
-        return base64.StdEncoding.EncodeToString(rnd)
+	rnd := make([]byte, 32)
```

Comment:
The formatting is controlled by gofmt, it formats all go code, all go projects use it